routes/users.js

```diff
@@ -33,7 +33,7 @@ router.patch(
   users.updateDiscordUserNickname
 );
 router.get("/:username", users.getUser);
-router.get("/:userId/intro", authenticate, authorizeRoles([SUPERUSER]), users.getUserIntro);
```

Comment:
Why this? 

As we already have  application route why you can't fetch with it

Reply:
Have to change many code in website-www, to use this application route. Also user's should see their application status in the `/intro` route only.

Reply:
But this is not the good solution 
And ask with Ankush or Tejas is your PR is going to merge or my

Reply:
And also in /intro route we are using /application API

Reply:
Only superusers would be able to accept new user request, no one else, also if you're not the applicants you will always get `you are not authorised `

Reply:
But there is already route present for view the application then what is the need of this new route?

Reply:
I don't build this from the ground up, it was there, just did some modifications, also previously told you that if I wanted to use the application route , will have to change many code.